### ISSUE TITLE
Memo: how fonts are handled in Sage Games

### ISSUE THREAD

**Issue by BSG-75** [OpenSAGE/OpenSAGE#405]
These are taken from Discord discussion between me, @Qibbi and @feliwir on this issue: https://github.com/OpenSAGE/OpenSAGE/issues/398
I'm posting them there so it's not lost in the chat history
```
Language9x.ini ... it seems Generals uses a postfix for their ini names which denotes that it has another encoding... 
seems the 9x postfix is only for Language and HeaderTemplate
***9x.ini is used up to BFME, 
BFME2 changed that to only use fontsubstitution.ini, RA3 has that stuff inside compiled data

RA3+ just calls AddFontResourceExA (from GDI32.dll) on all fonts present in data/fonts/
TW I'm not 100%

Arial just has a few simple codepages, only Arial Unicode has the unicode stuff and asian languages
Microsoft used to ship it with Microsoft Office, but stopped shipping it in recent versions of Office.

That could be the reason why Generals uses Times New Roman, TNR seems to be the fallback fallback font, that one's hardcoded into the engine.
basically the "if we cannot find a font we actually want to do Times New Roman"

maybe we want something like Code2000
it also supports Tengwar and Klingon!
```

It might also be a good idea to wait for this one for handling fallback fonts:
https://github.com/SixLabors/Fonts/issues/117
